### ISSUE TITLE
Is it possible to create a portable/standalone, cross-platform instance with all dependencies?

### ISSUE THREAD

**Issue by xeijin** [apache/incubator-superset#4069]
More of a question than an issue, but I'd like to be able to use Superset on projects at work (windows desktop, though cross-platform would be ideal). Since external package repositories are blocked by many corporate firewalls, I was hoping to create a standalone version with all of the relevant dependencies, upload to a Github repo and then simply download the zip and execute from the folder.

The main issue I foresee is the requirement for the cryptography python dependency, which, as mentioned in the documentation is an 'OS-level dependency' https://superset.incubator.apache.org/installation.html#os-dependencies

I realise something like Docker would be ideal for this use-case but unfortunately, I will not have the necessary administrator privileges to access the hypervisor.

**Comment by xrmx** [comment]
Whatever solution fits your problem it's out of scope of this project. There are projects creating installers for python apps, give them a try. Also friendly reminder that superset is not supported on windows.

**Comment by xeijin** [comment]
Yes, aware it is out of scope of the project, but hoping someone here may have overcome this in another way, or otherwise have other expertise to help point me in the right direction (please do let me know if there is a more appropriate place to ask questions such as these).

I did think about an installer, but as I do not have administrator privileges on work machines not sure that would work for me.

**Comment by mistercrunch** [comment]
![docker1](https://user-images.githubusercontent.com/487433/34082104-17e3aa58-e30d-11e7-883e-43a2948e5ffa.png)

**Comment by witwall** [comment]
btw，I just made a Windows Portable Version.

**Comment by xeijin** [comment]
So have I, using virtualenv and I have to say it was a real pain, python seems to make any kind of portability almost comically painful.

Docker doesn't work for me as it requires administrative privileges.

**Comment by witwall** [comment]
@xeijin 

 try [superset portable version](https://github.com/apache/incubator-superset)

**Comment by xeijin** [comment]
@witwall Can you explain how it works? Seems to link to the same installation instructions ... ?

**Comment by witwall** [comment]
@xeijin  sorry, and I updated the link now.
